README.md

```diff
@@ -1,24 +1,173 @@
-# README
+<br>
+<div align='center'>
+	<h1>Wise Wallet</h1>
+  </div>
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Demo](#demo)
+    - [Video Presentation](#video) 
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+<!-- PROJECT DESCRIPTION -->
 
-* System dependencies
+# 📖 [Wise wallet] <a name="about-project"></a>
 
-* Configuration
 
-* Database creation
+Effortlessly monitor and control your finances with the convenience of this web application. Streamline the categorization of your transactions and access valuable insights into your spending patterns. This empowers you to make well-informed financial decisions, ensuring effective budget management.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<!-- Features -->
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- Well usage of Gitflow
+- Register a user
+- Login registered user
+- Create categories and transactions
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀Live Demo <a name="demo"></a>
+
+[Live Demo](https://)
```

Comment:
- Kindly, as the previous reviwer mentioned that deploy your project to make it more accessible :+1:

Reply:
well resolved! ✔️